<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>441</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="bx_x">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>480</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="bx_y">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>480</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPorts">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstMessages">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>50</y>
      <width>431</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnMessage">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenPorts">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPorts">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
